Orange/data/io.py

```diff
@@ -369,7 +369,8 @@ def _equal_length(lst):
 
             if type_flag in StringVariable.TYPE_HEADERS:
                 coltype = StringVariable
-
+                values = [None if isinstance(i, float)
```

Comment:
Since the tab format is specified to count `\t\t` values as missing (and since Python's `csv` module doesn't handle `\t\t` and `\t""\t` differently), we can't and never have distinguished between missing values and true empty strings anyway.